builds/e2e/e2e.yaml

```diff
@@ -57,29 +70,122 @@ jobs:
     - template: templates/e2e-setup.yaml
     - template: templates/e2e-run.yaml
 
-################################################################################	
-  - job: windows_amd64
-################################################################################	
-    displayName: Windows amd64
+################################################################################
+  - job: ubuntu_1804_docker
+################################################################################
+    displayName: Ubuntu 18.04 with Docker (minimal)
 
     pool:
-      name: $(pool.name)
+      name: $(pool.linux.name)
       demands:
-        - Agent.OS -equals Windows_NT
-        - Agent.OSArchitecture -equals X64
-        - run-new-e2e-tests -equals true
+        - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
+
+    variables:
+      os: linux
+      arch: amd64
+      artifactName: iotedged-ubuntu18.04-amd64
+      minimal: true
+
+    steps:
+    - template: templates/e2e-setup.yaml
+    - template: templates/e2e-run.yaml
+
+################################################################################
+  - job: ubuntu_2004_msmoby
+################################################################################
+    displayName: Ubuntu 20.04 with iotedge-moby
+
+    pool:
+      name: $(pool.linux.name)
+      demands:
+        - ImageOverride -equals agent-aziotedge-ubuntu-20.04-msmoby
+
+    variables:
+      os: linux
+      arch: amd64
+      artifactName: iotedged-ubuntu18.04-amd64
```

Comment:
Do we have a plan to also have the edgelet build on ubuntu20.04? 
(Our sibling repro... IIS have artifact outputs built from ubuntu20.04).

Reply:
I don't think there's much of a difference between 18.04 and 20.04 from IoT Edge's perspective. All I can think of is a different version of openssl.

I think it would be easy to add 20.04 to packages.yaml, although I don't know the specifics. But that's probably the last piece required to officially support it. Didn't you already add it to the release process (but not enabled)?

Reply:
> I think it would be easy to add 20.04 to packages.yaml, although I don't know the specifics. But that's probably the last piece required to officially support it. Didn't you already add it to the release process (but not enabled)?

That's only for release/1.2. For release/1.2 branch, we have edgelet & IIS built for Ubuntu20.04. The pipeline modification was done by Core's team... I think if we get this PR into release/1.2 branch, we maybe able to have the automated test for ubuntu 20.04 and once the Connectivity & LH tests become available for 1ES there, we can make Ubuntu20.04 as tier1.

Per release/1.1,  I'm not sure what is our strategy regarding Ubuntu20.04. @micahl ?

Reply:
@yophilav we want to make Ubuntu 20.04 Tier 1 (for both the LTS and 1.2+) by supporting it in our test pipeline and then publishing the respective packages.  We've delayed doing this until we're able to include it in our test matrix.